packer/fah_ami.json

```diff
@@ -0,0 +1,46 @@
+{
+  "variables": {
+    "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
+    "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
+    "region": "{{env `AWS_DEFAULT_REGION`}}",
+    "fah_team": "{{user `fah_team`}}",
+    "fah_user": "{{user `fah_user`}}",
+    "fah_passkey": "{{user `fah_passkey`}}"
+  },
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "ami_name": "packer-ubuntu-folding-at-home-{{timestamp}}",
+      "instance_type": "c5n.large",
+      "access_key": "{{user `aws_access_key`}}",
+      "secret_key": "{{user `aws_secret_key`}}",
+      "region": "{{user `region`}}",
+      "source_ami_filter": {
+        "filters": {
+          "virtualization-type": "hvm",
+          "name": "ubuntu/images/*ubuntu-xenial-16.04-amd64-server-*",
```

Comment:
any specific reasons for Ubuntu 16.04? If not 20, we could use 18 - of course, this would be up to users to change if needed but nice defaults are nice

Reply:
Force of habit I guess? We also know 16 and 18 work, lets do 20 after a bit of testing? Bumping this up to 18.04 for now because that is the better choice.